app/src/main/kotlin/com/wire/android/ui/common/UserProfileAvatar.kt

```diff
@@ -56,7 +56,7 @@ fun UserProfileAvatar(
                 .background(Color.Black, CircleShape)
                 .size(size)
                 .clip(CircleShape),
-            contentScale = ContentScale.FillBounds,
+            contentScale = ContentScale.FillWidth,
```

Comment:
According to the latest design requirements, this will need to be changed programmatically depending on the image to upload exif information:
_Depending on whether the image is in portrait or landscape mode, given by its Exif information, we should apply a content scale filter of FillWidth or FillHeight_

Reply:
I spend a couple of hours trying to implement this dynamic scale filter, but had to implement another solution using `ContentScale.Crop` (This will Scale the source uniformly (maintaining the source's aspect ratio)). 

Having the Exif data loaded it's a bit overkill and personally have concerns about the performance, since we have to open `InputStream`s, or open Files to extract this metadata. I was thinking on how this will affect performance, for instance in a group conversation loading every avatar, having to calculate the Exif data for every user.

Using `ContentScale.Crop` solves the issue for both Landscape and Portrait, achieving images to maintain aspect ratio 🚀  and we are doing the Exif compute just once, only when we save/upload the avatar